src/main/java/uk/gov/hmcts/dts/fact/services/MapitService.java

```diff
@@ -55,4 +55,21 @@ public MapitService(final Logger logger, final MapitClient mapitClient) {
 
         return Optional.empty();
     }
+
+    public Boolean localAuthorityExists(final String localAuthorityName) {
+
+        if (StringUtils.isNotBlank(localAuthorityName)) {
+            try {
+                return mapitClient.getMapitDataForLocalAuthorities(localAuthorityName)
+                    .values()
+                    .stream()
+                    .anyMatch(la -> la.getName().equalsIgnoreCase(localAuthorityName));
+            } catch (final FeignException ex) {
+                logger.warn("HTTP Status: {} Message: {}", ex.status(), ex.getMessage(), ex);
```

Comment:
This is going to be logged into the file. Is the exception message returned mentions the error happens when using mapIt to search for local authority? If not, it can be confusing as someone looking at the log file might think it is an error for something else.

Reply:
I've added some extra info into the logging to ensure it's clear that it's a mapit API call that's failed.